tests/end2end/features/misc.feature

```diff
@@ -671,3 +671,38 @@ Feature: Various utility commands.
         And I wait until the scroll position changed
         And I run :click-element id link
         Then the error "Element position is out of view!" should be shown
+
+    ## :command-history-{prev,next}
+    
+    Scenario: Calling previous command
+        When I run :set-cmd-text :message-info blah
+        And I run :command-accept
+        And I wait for "blah" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-accept
+        Then the message "blah" should be shown
+    
+    Scenario: Browsing through commands 
+        When I run :set-cmd-text :message-info blarg
+        And I run :command-accept
+        And I wait for "blarg" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-history-prev
```

Comment:
This test will probably fail when run in isolation, as only one command has been run via the commandline, correct? It'd probably be a good idea to add another command before the one showing "blarg".